package.json

```diff
@@ -45,7 +45,7 @@
     "mocha": "^3.3.0",
     "source-map-support": "^0.4.15",
     "tslint": "^5.1.0",
-    "typescript": "^2.3.1"
```

Comment:
This version of TypeScript is way to old.  Updating it resulted in a few new type errors, but nothing a few casts and `@ts-expect-error`s couldn't handle.